src/exchange/WebSocket.ts

```diff
@@ -117,6 +118,46 @@ export class WebSockets {
     return true;
   }
 
+  subscribeOrderBookDepthStreamBySymbol(
+    symbol: MarketSymbol,
+    depth = ""
+  ): boolean {
+    if (!this.socketInstance) return false;
+    this.socketInstance.send(
+      JSON.stringify([
+        "SUBSCRIBE",
+        [
+          {
+            e: SOCKET_EVENTS.ORDERBOOK_DEPTH_STREAM_ROOM,
+            p: symbol,
+            d: depth,
```

Comment:
What would happen if depth is empty string, would it subscribe to all depth levels ?

Reply:
I can see in the [dapi PR](https://github.com/fireflyprotocol/dapi-server/pull/1198/files) it defaults to 5. @NidaKarimAli is it not meant to be 20 by default?

Reply:
I also think we should provide the default value into the client as well as anyone using the client would not know what the default is

Same for the other clients

Reply:
Discussed async on the above comments.